apps/stripe/src/modules/saleor/transaction-event-reporter.ts

```diff
@@ -69,12 +72,32 @@ export class TransactionEventReporter implements ITransactionEventReporter {
       const mutationResult = await this.gqlClient.mutation(TransactionEventReportDocument, {
         ...input,
         amount: input.amount.amount,
+        availableActions: input.actions,
       });
 
       const { data, error } = mutationResult;
 
       if (error) {
-        switch (error.cause as TransactionEventReportErrorCode) {
+        return err(
+          new UnhandledError("Error reporting transaction event - server error", {
+            cause: error,
+          }),
+        );
+      }
+
+      const mutationErrors = data?.transactionEventReport?.errors ?? [];
+
+      const hasMoreThanOneError = mutationErrors.length > 1;
+
+      if (hasMoreThanOneError) {
+        this.logger.warn(
+          "TransactionEventReport mutation has more than one error - handling the first one",
+          mutationErrors,
+        );
+      }
+
+      if (mutationErrors.length > 0) {
```

Comment:
changed how we handle errors from Saleor - errors are inside `data.transactionEventReport.errors` and are not merged into Urql `errors`